tutorials/polkadot-sdk/system-chains/asset-hub/send-a-tx-paying-fee-different-token.md

```diff
@@ -0,0 +1,167 @@
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:1:12"
+```
+
+### Define Constants
+
+Define the constants for your transaction:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:14:16"
+```
+
+### Create a Signer
+
+Create a signer using Alice's development account:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:18:29"
+```
+
+This function will return a signer that can be used to sign the transaction.
+
+### Setup the Client and API
+
+Create the client connection to the local Asset Hub:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:31:37"
+```
+
+### Create the Transaction
+
+Create a standard DOT transfer transaction:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:39:42"
+```
+
+This creates a transaction that transfers 3 DOT to Bob's address while keeping Alice's account alive.
+
+### Sign and Submit with Alternative Fee Payment
+
+The key part of this tutorial is specifying an alternative asset for fee payment. This is done through the `asset` parameter in the `signAndSubmit` options:
+
+```typescript title="fee-payment-transaction.ts"
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts:44:69"
+```
+
+This specifies that the fees should be paid using the USDT asset.
+
+## Full Code
+
+The full code for the complete implementation is the following:
+
+??? code "Complete Code"
+
+    ```typescript title="fee-payment-transaction.ts"
+    --8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction.ts"
+    ```
+
+## Running the Script
+
+To run the script:
+
+```bash
+npx ts-node fee-payment-transaction.ts
+```
+
+## Expected Output
+
+When you run the script successfully, you should see output similar to:
+
+--8<-- "code/tutorials/polkadot-sdk/parachains/system-chains/asset-hub/send-tx-paying-fees-with-different-tokens/fee-payment-transaction-output.html"
+
+The key events to look for are:
+
+- **Assets**: The asset was transferred.
+- **Balances**: The fees were paid using the alternative asset.
+- **AssetConversion**: The fees were converted to the alternative asset.
+- **AssetTxPayment**: The fees were paid using the alternative asset.
+- **System**: The transaction was successful.
```

Comment:
Are these code elements? Do they need to be in backticks? Like `AssetConversion` looks like it does